{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "4cc0e443",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Generated metadata file noaa-aoml-ngmt_18S-V9_metadata.tsv\n",
      "Generated metadata file noaa-aoml-ngmt_16S-V4V5_metadata.tsv\n",
      "Generated manifest file 20220912_16S-Amplicon_PE250_16S-V4V5_manifest.tsv\n",
      "Generated manifest file 20231110_AND14513_Amplicon_PE250_16S-V4V5_manifest.tsv\n",
      "Generated manifest file 20240325_16S-Amplicon_PE250_16S-V4V5_manifest.tsv\n",
      "Generated manifest file 20250423_16S-Amplicon_PE250_16S-V4V5_manifest.tsv\n",
      "Generated manifest file 20250728_THO16883_16S-Amplicon_PE250_16S-V4V5_manifest.tsv\n",
      "Generated manifest file 20220916_18S-Amplicon_PE250_18S-V9_manifest.tsv\n",
      "Generated manifest file 20231107_AND14514_Amplicon_PE150_18S-V9_manifest.tsv\n",
      "Generated manifest file 20240401_18S-Amplicon_PE150_18S-V9_manifest.tsv\n",
      "Generated manifest file 20250423_18S-Amplicon_PE150_18S-V9_manifest.tsv\n",
      "Generated manifest file 20250730_THO16884_18S-Amplicon_PE150_18S-V9_manifest.tsv\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import argparse\n",
    "import os\n",
    "\n",
    "pd.set_option('future.no_silent_downcasting', True)\n",
    "\n",
    "path_faire = '~/sedtrap/metadata/FAIRe-NOAA_noaa-aoml-ngmt_20260109.xlsx'\n",
    "absolute_path_sequences = '/data/sequences/MichiganState'\n",
    "output_directory = '.'\n",
    "column_suffix = 'pcr_plate_id'\n",
    "direction_suffix = 'right'\n",
    "num_chars_suffix = 8\n",
    "delimiter_suffix = '_'\n",
    "         \n",
    "df_project = pd.read_excel(path_faire, sheet_name='projectMetadata', comment='#')\n",
    "df_sample = pd.read_excel(path_faire, sheet_name='sampleMetadata', comment='#')\n",
    "df_exptrun = pd.read_excel(path_faire, sheet_name='experimentRunMetadata', comment='#')\n",
    "\n",
    "df_project_first_2_cols_removed = df_project.iloc[:, 2:]\n",
    "df_project_first_2_cols_removed.set_index('term_name', inplace=True)\n",
    "df_project_wide = df_project_first_2_cols_removed.transpose()\n",
    "project_id = df_project_wide['project_id'].iloc[0]\n",
    "first_row_values = df_project_wide.loc['project_level']\n",
    "df_project_filled = df_project_wide.fillna(first_row_values)\n",
    "\n",
    "seq_run_ids = df_exptrun['seq_run_id'].unique()\n",
    "assay_names = df_exptrun['assay_name'].unique()\n",
    "if set(assay_names) != set(df_exptrun['assay_name'].dropna()):\n",
    "    raise ValueError(\"Inconsistent assay names between projectMetadata and experimentRunMetadata.\")\n",
    "\n",
    "# Metadata file for each assay_name\n",
    "dict_assay_short = {}\n",
    "for assay_name in df_exptrun['assay_name'].unique()[::-1]:\n",
    "    # For each assay, merge experiment run metadata with sample metadata and project metadata\n",
    "    metadata = df_exptrun[df_exptrun['assay_name'] == assay_name].merge(df_sample, on='samp_name', how='left', suffixes=('', '_SAMPLE'))\n",
    "    metadata = pd.merge(metadata, df_project_filled.loc[assay_name].to_frame().transpose(), how='cross', suffixes=('', '_PROJECT'))\n",
    "    metadata.dropna(axis=1, how='all', inplace=True)\n",
    "    metadata.rename(columns={'samp_name': 'sample_name'}, inplace=True)\n",
    "    \n",
    "    # Add suffix to sample names that are duplicated within the assay using column_suffix (if provided), direction_suffix, num_chars_suffix, and delimiter_suffix\n",
    "    if column_suffix is not None:\n",
    "        duplicated_samples = metadata['sample_name'][metadata['sample_name'].duplicated(keep=False)].unique()\n",
    "        for sample in duplicated_samples:\n",
    "            sample_rows = metadata[metadata['sample_name'] == sample]\n",
    "            for idx, row in sample_rows.iterrows():\n",
    "                if direction_suffix == 'right':\n",
    "                    suffix = f'{delimiter_suffix}{row[column_suffix][-num_chars_suffix:]}'  # Placeholder suffix\n",
    "                else:               \n",
    "                    suffix = f'{delimiter_suffix}{row[column_suffix][:num_chars_suffix]}'  # Placeholder suffix\n",
    "                #new_sample_name = f\"{row['sample_name']}{suffix}\"\n",
    "                #metadata.at[idx, 'sample_name'] = new_sample_name\n",
    "    \n",
    "    # Short assay name to use for output files\n",
    "    gene = df_project_filled.loc[assay_name]['target_gene'].split(' ')[0]\n",
    "    subfragment = df_project_filled.loc[assay_name]['target_subfragment']\n",
    "    subfragment_part = subfragment.split(' ')[0].replace('-', '').replace('_', '') if isinstance(subfragment, str) and subfragment.strip() else ''\n",
    "    if subfragment_part:\n",
    "        dict_assay_short[assay_name] = f\"{gene}-{subfragment_part}\"\n",
    "    else:\n",
    "        dict_assay_short[assay_name] = gene\n",
    "    metadata.to_csv(os.path.join(output_directory, f\"{project_id}_{dict_assay_short[assay_name]}_metadata.tsv\"), sep='\\t', index=False)\n",
    "    print(f\"Generated metadata file {project_id}_{dict_assay_short[assay_name]}_metadata.tsv\")\n",
    "\n",
    "# Manifest file for each seq_run_id\n",
    "for seq_run_id in seq_run_ids:\n",
    "    for assay_name in df_exptrun[df_exptrun['seq_run_id'] == seq_run_id]['assay_name'].unique():\n",
    "        manifest = df_exptrun[(df_exptrun['seq_run_id'] == seq_run_id) & (df_exptrun['assay_name'] == assay_name)][['samp_name', 'filename', 'filename2']]\n",
    "        manifest['filename'] = manifest['filename'].apply(lambda x: os.path.join(absolute_path_sequences, f'{seq_run_id}', str(x)))\n",
    "        manifest['filename2'] = manifest['filename2'].apply(lambda x: os.path.join(absolute_path_sequences, f'{seq_run_id}', str(x)))\n",
    "        manifest.columns = ['sample-id', 'forward-absolute-filepath', 'reverse-absolute-filepath']\n",
    "        manifest.to_csv(os.path.join(output_directory, f\"{seq_run_id}_{dict_assay_short[assay_name]}_manifest.tsv\"), sep='\\t', index=False)\n",
    "        print(f\"Generated manifest file {seq_run_id}_{dict_assay_short[assay_name]}_manifest.tsv\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "abcef8c4",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'_20250626'"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "suffix"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "bd58aaed",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "sample_name",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "assay_name",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pcr_plate_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pcr_well_position",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pcr_well_number",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "lib_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "seq_run_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "filename",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "filename2",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "checksum_filename",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "checksum_filename2",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "serial_number",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "samp_category",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "neg_cont_type",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pos_cont_type",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "cruise_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "station",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "ctd_bottle_no",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "sample_replicate",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "notes_sampling",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "ncei_doi",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "ncei_sample_id",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "materialSampleID",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "sample_derived_from",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "sample_composed_of",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "biological_rep_relation",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "extraction_plate_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "extraction_well_number",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "extraction_well_position",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "extraction_date",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "decimalLongitude",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "decimalLatitude",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "verbatimLongitude",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "verbatimLatitude",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "verbatimCoordinateSystem",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "verbatimSRS",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "stationLatitude",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "stationLongitude",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "geo_loc_name",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "eventDate",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "eventDateStart",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "eventDateMidpoint",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "eventDateEnd",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "eventDurationValue",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "verbatimEventDate",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "verbatimEventTime",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "durationTrapClosed",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "env_broad_scale",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "env_local_scale",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "env_medium",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "habitat_natural_artificial_0_1",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "samp_collect_method",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "samp_collect_device",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "samp_size",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "samp_size_unit",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "samp_store_temp",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "samp_store_sol",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "samp_store_dur",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "samp_store_loc",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "dna_store_loc",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "samp_store_method_additional",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "samp_mat_process",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "filter_passive_active_0_1",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "stationed_sample_dur",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pump_flow_rate",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pump_flow_rate_unit",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "size_frac",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "filter_material",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "filter_name",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "precip_chem_prep",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "precip_force_prep",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "precip_time_prep",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "precip_temp_prep",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "prepped_samp_store_temp",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "prepped_samp_store_sol",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "prepped_samp_store_dur",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "swimmers_0_1",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "swimmers_type",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "swimmers_notes",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "date_ext",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "samp_vol_we_dna_ext",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "samp_vol_we_dna_ext_unit",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "nucl_acid_ext_lysis",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "nucl_acid_ext_sep",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "nucl_acid_ext",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "nucl_acid_ext_kit",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "nucl_acid_ext_modify",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "dna_cleanup_0_1",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "dna_cleanup_method",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "concentration",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "concentration_unit",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "concentration_method",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "elution_vol",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "elution_vol_unit",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "dna_yield",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "dna_yield_unit",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pool_dna_num",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "assay_name_SAMPLE",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "minimumDepthInMeters",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "maximumDepthInMeters",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "tot_depth_water_col",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "temp",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "ph",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "ph_meth",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "salinity",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "total_alkalinty",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "total_alkalinty_AOML",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "diss_inorg_carb",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "diss_inorg_carb_unit",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "diss_inorg_carb_AOML",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "nitrate",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "nitrate_unit",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "nitrite",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "nitrite_unit",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "nitrate_and_nitrite",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "ammonium",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "phosphate",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "silicate",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "tot_mass_flux",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "part_org_carb_flux",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "part_inorg_carb_flux",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "part_opal_flux",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "terrig_flux",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "part_org_nitro_flux",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "Unnamed: 169",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "recordedBy",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "recordedByID",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "project_contact",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "institution",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "institutionID",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "project_name",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "project_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "study_factor",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "assay_type",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "sterilise_method",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "neg_cont_0_1",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pos_cont_0_1",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "checkls_ver",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "mod_date",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "license",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "rightsHolder",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "accessRights",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "code_repo",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pcr_0_1",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "inhibition_check_0_1",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "inhibition_check",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "thermocycler",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "assay_name_PROJECT",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "assay_validation",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "targetTaxonomicAssay",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "targetTaxonomicScope",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "target_gene",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "target_subfragment",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "ampliconSize",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pcr_primer_forward",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pcr_primer_reverse",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pcr_primer_name_forward",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pcr_primer_name_reverse",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pcr_primer_reference_forward",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pcr_primer_reference_reverse",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pcr_primer_vol_forward",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pcr_primer_vol_reverse",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pcr_primer_conc_forward",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pcr_primer_conc_reverse",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "probeReporter",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "probeQuencher",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "probe_seq",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "probe_ref",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "probe_conc",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "commercial_mm",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "custom_mm",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "block_seq",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "block_ref",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "block_taxa",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "amplificationReactionVolume",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pcr_dna_vol",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pcr_rep",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "nucl_acid_amp",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pcr_cond",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "annealingTemp",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pcr_cycles",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pcr_analysis_software",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pcr_method_additional",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "barcoding_pcr_appr",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pcr2_thermocycler",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pcr2_amplificationReactionVolume",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pcr2_commercial_mm",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pcr2_custom_mm",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pcr2_dna_vol",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pcr2_cond",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pcr2_annealingTemp",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pcr2_cycles",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pcr2_analysis_software",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pcr2_plate_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pcr2_method_additional",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "sequencing_location",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "platform",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "instrument",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "seq_kit",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "adapter_forward",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "adapter_reverse",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "lib_screen",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "checksum_method",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "seq_method_additional",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "ship_crs_expocode",
         "rawType": "object",
         "type": "string"
        }
       ],
       "ref": "3c632e6d-e9f4-490d-8bab-3e84059dbf1c",
       "rows": [
        [
         "27",
         "NGMT_GMT1_Sed_550m_10A",
         "Bacteria-16S-V4V5-Parada",
         "GMT-1 Sed 16S 2022-09",
         "D4",
         "28",
         "GMT1_16S_10A_S28",
         "20220912_16S-Amplicon_PE250",
         "GMT1_16S_10A_S28_L001_R1_001.fastq.gz",
         "GMT1_16S_10A_S28_L001_R2_001.fastq.gz",
         "d678bd9054c9e861f2e21f3b19f050d7",
         "24283280b730f5ca4184559a45667076",
         "GMT1_10A",
         "sample",
         "not applicable",
         "not applicable",
         "GMT-1",
         "SW01",
         "not applicable",
         "A",
         "not applicable",
         null,
         null,
         "NGMT_GMT1_Sed_550m_10",
         "NGMT_GMT1_Sed_550m_10",
         "not applicable",
         "NGMT_GMT1_Sed_550m_10B | NGMT_GMT1_Sed_550m_10C",
         "GMT1_Plate1",
         "28",
         "D4",
         "2022-08-05 00:00:00",
         "-89.1180000000001",
         "28.2399999999999",
         "89°7.100'W",
         "28°14.387'N",
         "DDM",
         "WGS84",
         "28",
         "-89",
         "USA: Gulf of Mexico",
         "2022-03-29/2022-04-12",
         "2022-03-29",
         "2022-04-05 00:00:00",
         "2022-04-12 00:00:00",
         "P14D",
         "29-Mar-2022/12-Apr-2022",
         "not applicable",
         "81",
         "marginal sea biome (ENVO:01000046)",
         "marine aphotic zone (ENVO:00000210)",
         "marine mud (ENVO:00005795)",
         "0",
         null,
         "sediment trap",
         "0.291",
         "mg",
         "4",
         "other: formalin",
         "P3M24D",
         "AOML Room 248, 4° refridgerator",
         "AOML Room 248, -20° freezer",
         "Sediment samples were stored in a refridgerator at 4°C with DI water covering the sediment before eDNA extraction.",
         null,
         "not applicable",
         null,
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "Solution CD2",
         "15000",
         "1",
         "25",
         "4",
         "other: formalin",
         "not applicable",
         "0",
         "not applicable",
         "not applicable",
         "2022-08-05 00:00:00",
         "0.291",
         "g",
         "chemical",
         "magnetic beads",
         "https://www.qiagen.com/us/resources/resourcedetail?id=5a0517a7-711d-4085-8a28-2bb25fab828a&lang=en",
         "Qiagen PowerSoil Kit",
         "https://github.com/aomlomics/protocol_extract_dna_sedtrap_powersoil/blob/main/protocol_extract_dna_sedtrap_powersoil.md",
         "1",
         "NEBNext DNA Repair",
         "16.68",
         "ng/µl",
         "Quant-iT 1X dsDNA Assay Kit (Invitrogen)",
         "32",
         "uL",
         "533.76",
         "ng",
         "1",
         "Bacteria-16S-V4V5-Parada | Eukarya-18S-V9-Lane-Medlin",
         "550",
         "550",
         "1100",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "1029.94",
         "41.05",
         "110.2",
         "149.85",
         "667.26",
         "5.85",
         null,
         "Luke Thompson | Emily Osborne | Adrianne Wilson | Gabriella Lirio | Sammy Harding",
         "https://orcid.org/0000-0002-3911-1280 | https://orcid.org/0000-0001-9579-5851 | https://orcid.org/0009-0006-1171-6014 | https://orcid.org/0009-0000-5574-1349 | https://orcid.org/0009-0008-8885-6140",
         "luke.thompson@noaa.gov",
         "NOAA Atlantic Oceanographic and Meteorological Laboratories | NOAA Ocean Acidification Program",
         "https://ror.org/042r9xb17 | https://ror.org/02bfn4816",
         "Northern Gulf of Mexico Sediment Trap",
         "noaa-aoml-ngmt",
         "sediment trap time series | water column time series",
         "metabarcoding",
         "After sampling, run ~1 L of 5% bleach through tubing lines, then repeat with ~1 L of deionized water. Rinse the 20-L collection carboy 3 times with ~0.5 L DI water (1.5 L total) after sampling. Before the next sample, rinse 20-L carboy 2 times with ~100 mL of sample water (200 mL total).",
         "1",
         "1",
         "FAIRe_NOAA_checklist_v1.0.2.xlsx",
         "2025-11-12 00:00:00",
         "https://creativecommons.org/publicdomain/zero/1.0/",
         "US Government",
         "No Rights Reserved",
         "https://github.com/aomlomics/sedtrap",
         "1",
         "0",
         "not applicable",
         "Eppendorf Mastercycler Nexus Thermal Cycler",
         "Bacteria-16S-V4V5-Parada",
         "not provided",
         "16S rRNA gene sequencing targeting the V4-V5 region using primers 515F and 926R",
         "Prokaryotes, including Bacteria and Archaea",
         "16S rRNA (SSU prokaryote)",
         "V4-V5",
         "411",
         "GTGYCAGCMGCCGCGGTAA",
         "CCGYCAATTYMTTTRAGTTT",
         "515F-Y",
         "926R",
         "10.1111/1462-2920.13023",
         "10.1111/1462-2920.13023",
         "1",
         "1",
         "10",
         "10",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "AmpliTaq Gold 360 Master Mix, Thermo Fisher Scientific",
         "PCR reactions were run in 25 uL reaction volumes, with 1.0 uL of DNA, 12.5 uL of AmpliTaq Gold, 9.5 uL of water, and 1.0 uL of each primer (10 uM).",
         "not applicable",
         "not applicable",
         "not applicable",
         "25",
         "1",
         "not applicable",
         "https://doi.org/10.1111/1462-2920.13023",
         "initial denaturation:95_3;denaturation:95_0.75;annealing:50_0.75;elongation:72_1.5;final elongation:72_10;30",
         "50",
         "30",
         "not applicable",
         "not applicable",
         "two-step PCR",
         "not provided",
         "15",
         "DreamTaq Hot Start PCR Master Mix (K9012), Thermo Fisher Scientific",
         "not applicable",
         "1",
         "initial denaturation:95_3;denaturation:95_0.25;annealing:60_0.5;elongation:72_1;final elongation:72_3;15",
         "60",
         "11",
         "not applicable",
         "not provided",
         "not applicable",
         "Michigan State University Research Technology Support Facility Genomics Core",
         "ILLUMINA",
         "Illumina MiSeq [OBI_0002003]",
         "MiSeq v2 500",
         "ACACTGACGACATGGTTCTACA",
         "TACGGTAGCAGAGACTTGGTCT",
         "The Genomics Core performed secondary PCR using dual indexed, Illumina-compatible primers targeting the Fluidigm CS1/CS2 oligomers at the ends of the primary PCR products. Amplicons were batch normalized using Invitrogen SequalPrep DNA Normalization plates and recovered product was pooled. The pool was QC’d and quantified using a combination of Qubit dsDNA HS, Agilent 4200 TapeStation HS DNA1000, and Invitrogen Collibri Library Quantification qPCR assays.",
         "MD5",
         "PhiX control library was spiked in at 10%.",
         "32PE | 320G"
        ],
        [
         "49",
         "NGMT_GMT1_Sed_550m_10A_Extracted2306",
         "Bacteria-16S-V4V5-Parada",
         "GMT-2 Sed 16S 2023-08",
         "D1",
         "4",
         "GMT1_dup_16S_10A_S4",
         "20231110_AND14513_Amplicon_PE250",
         "GMT1_dup_16S_10A_S4_L001_R1_001.fastq.gz",
         "GMT1_dup_16S_10A_S4_L001_R2_001.fastq.gz",
         "3c544b57b6949fd3ca628f2596c03dc9",
         "2b7ec642895560b4327365eed3a3916f",
         "GMT1_10A_dup",
         "sample",
         "not applicable",
         "not applicable",
         "GMT-1",
         "SW01",
         "not applicable",
         "A",
         "re extracted GMT-1 samples (extracted with GMT-2 samples, June 2023)",
         null,
         null,
         "NGMT_GMT1_Sed_550m_10",
         "NGMT_GMT1_Sed_550m_10",
         "not applicable",
         "NGMT_GMT1_Sed_550m_10B_dup | NGMT_GMT1_Sed_550m_10C_dup",
         "GMT-2 Sed DNA",
         "4",
         "D1",
         "2023-06-27 00:00:00",
         "-89.1180000000002",
         "28.2399999999999",
         "89°7.100'W",
         "28°14.387'N",
         "DDM",
         "WGS84",
         "28",
         "-89",
         "USA: Gulf of Mexico",
         "2022-03-29/2022-04-12",
         "2022-03-29",
         "2022-04-05 00:00:00",
         "2022-04-12 00:00:00",
         "P14D",
         "29-Mar-2022/12-Apr-2022",
         "not applicable",
         "407",
         "marginal sea biome (ENVO:01000046)",
         "marine aphotic zone (ENVO:00000210)",
         "marine mud (ENVO:00005795)",
         "0",
         null,
         "sediment trap",
         "0.297",
         "mg",
         "4",
         "other: formalin",
         "P1Y2M15D",
         "AOML Room 248, 4° refridgerator",
         "AOML Room 248, -20° freezer",
         "Sediment samples were stored in a refridgerator at 4°C with DI water covering the sediment before eDNA extraction.",
         null,
         "not applicable",
         null,
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "Solution CD2",
         "15000",
         "1",
         "25",
         "4",
         "other: formalin",
         "not applicable",
         "0",
         "not applicable",
         "not applicable",
         "2023-06-27 00:00:00",
         "0.297",
         "g",
         "chemical",
         "magnetic beads",
         "https://www.qiagen.com/us/resources/resourcedetail?id=5a0517a7-711d-4085-8a28-2bb25fab828a&lang=en",
         "Qiagen PowerSoil Kit",
         "https://github.com/aomlomics/protocol_extract_dna_sedtrap_powersoil/blob/main/protocol_extract_dna_sedtrap_powersoil.md",
         "1",
         "NEBNext DNA Repair",
         "4.8015",
         "ng/µl",
         "Quant-iT 1X dsDNA Assay Kit (Invitrogen)",
         "32",
         "uL",
         "153.648",
         "ng",
         "1",
         "Bacteria-16S-V4V5-Parada | Eukarya-18S-V9-Lane-Medlin",
         "550",
         "550",
         "1100",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "1029.94",
         "41.05",
         "110.2",
         "149.85",
         "667.26",
         "5.85",
         null,
         "Luke Thompson | Emily Osborne | Adrianne Wilson | Gabriella Lirio | Sammy Harding",
         "https://orcid.org/0000-0002-3911-1280 | https://orcid.org/0000-0001-9579-5851 | https://orcid.org/0009-0006-1171-6014 | https://orcid.org/0009-0000-5574-1349 | https://orcid.org/0009-0008-8885-6140",
         "luke.thompson@noaa.gov",
         "NOAA Atlantic Oceanographic and Meteorological Laboratories | NOAA Ocean Acidification Program",
         "https://ror.org/042r9xb17 | https://ror.org/02bfn4816",
         "Northern Gulf of Mexico Sediment Trap",
         "noaa-aoml-ngmt",
         "sediment trap time series | water column time series",
         "metabarcoding",
         "After sampling, run ~1 L of 5% bleach through tubing lines, then repeat with ~1 L of deionized water. Rinse the 20-L collection carboy 3 times with ~0.5 L DI water (1.5 L total) after sampling. Before the next sample, rinse 20-L carboy 2 times with ~100 mL of sample water (200 mL total).",
         "1",
         "1",
         "FAIRe_NOAA_checklist_v1.0.2.xlsx",
         "2025-11-12 00:00:00",
         "https://creativecommons.org/publicdomain/zero/1.0/",
         "US Government",
         "No Rights Reserved",
         "https://github.com/aomlomics/sedtrap",
         "1",
         "0",
         "not applicable",
         "Eppendorf Mastercycler Nexus Thermal Cycler",
         "Bacteria-16S-V4V5-Parada",
         "not provided",
         "16S rRNA gene sequencing targeting the V4-V5 region using primers 515F and 926R",
         "Prokaryotes, including Bacteria and Archaea",
         "16S rRNA (SSU prokaryote)",
         "V4-V5",
         "411",
         "GTGYCAGCMGCCGCGGTAA",
         "CCGYCAATTYMTTTRAGTTT",
         "515F-Y",
         "926R",
         "10.1111/1462-2920.13023",
         "10.1111/1462-2920.13023",
         "1",
         "1",
         "10",
         "10",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "AmpliTaq Gold 360 Master Mix, Thermo Fisher Scientific",
         "PCR reactions were run in 25 uL reaction volumes, with 1.0 uL of DNA, 12.5 uL of AmpliTaq Gold, 9.5 uL of water, and 1.0 uL of each primer (10 uM).",
         "not applicable",
         "not applicable",
         "not applicable",
         "25",
         "1",
         "not applicable",
         "https://doi.org/10.1111/1462-2920.13023",
         "initial denaturation:95_3;denaturation:95_0.75;annealing:50_0.75;elongation:72_1.5;final elongation:72_10;30",
         "50",
         "30",
         "not applicable",
         "not applicable",
         "two-step PCR",
         "not provided",
         "15",
         "DreamTaq Hot Start PCR Master Mix (K9012), Thermo Fisher Scientific",
         "not applicable",
         "1",
         "initial denaturation:95_3;denaturation:95_0.25;annealing:60_0.5;elongation:72_1;final elongation:72_3;15",
         "60",
         "11",
         "not applicable",
         "not provided",
         "not applicable",
         "Michigan State University Research Technology Support Facility Genomics Core",
         "ILLUMINA",
         "Illumina MiSeq [OBI_0002003]",
         "MiSeq v2 500",
         "ACACTGACGACATGGTTCTACA",
         "TACGGTAGCAGAGACTTGGTCT",
         "The Genomics Core performed secondary PCR using dual indexed, Illumina-compatible primers targeting the Fluidigm CS1/CS2 oligomers at the ends of the primary PCR products. Amplicons were batch normalized using Invitrogen SequalPrep DNA Normalization plates and recovered product was pooled. The pool was QC’d and quantified using a combination of Qubit dsDNA HS, Agilent 4200 TapeStation HS DNA1000, and Invitrogen Collibri Library Quantification qPCR assays.",
         "MD5",
         "PhiX control library was spiked in at 10%.",
         "32PE | 320G"
        ],
        [
         "50",
         "NGMT_GMT1_Sed_550m_10A_Extracted2306",
         "Bacteria-16S-V4V5-Parada",
         "GMT-3 Sed 16S 2023-07-28",
         "F6",
         "46",
         "GMT1_dup2_16S_10A_S91",
         "20231110_AND14513_Amplicon_PE250",
         "GMT1_dup2_16S_10A_S91_L001_R1_001.fastq.gz",
         "GMT1_dup2_16S_10A_S91_L001_R2_001.fastq.gz",
         "4c762a1dafa1b6094495bf1c9ebcc76f",
         "6170c9e0c03eab1bb1fc4447302dd978",
         "GMT1_10A_dup",
         "sample",
         "not applicable",
         "not applicable",
         "GMT-1",
         "SW01",
         "not applicable",
         "A",
         "re extracted GMT-1 samples (extracted with GMT-2 samples, June 2023)",
         null,
         null,
         "NGMT_GMT1_Sed_550m_10",
         "NGMT_GMT1_Sed_550m_10",
         "not applicable",
         "NGMT_GMT1_Sed_550m_10B_dup | NGMT_GMT1_Sed_550m_10C_dup",
         "GMT-2 Sed DNA",
         "4",
         "D1",
         "2023-06-27 00:00:00",
         "-89.1180000000002",
         "28.2399999999999",
         "89°7.100'W",
         "28°14.387'N",
         "DDM",
         "WGS84",
         "28",
         "-89",
         "USA: Gulf of Mexico",
         "2022-03-29/2022-04-12",
         "2022-03-29",
         "2022-04-05 00:00:00",
         "2022-04-12 00:00:00",
         "P14D",
         "29-Mar-2022/12-Apr-2022",
         "not applicable",
         "407",
         "marginal sea biome (ENVO:01000046)",
         "marine aphotic zone (ENVO:00000210)",
         "marine mud (ENVO:00005795)",
         "0",
         null,
         "sediment trap",
         "0.297",
         "mg",
         "4",
         "other: formalin",
         "P1Y2M15D",
         "AOML Room 248, 4° refridgerator",
         "AOML Room 248, -20° freezer",
         "Sediment samples were stored in a refridgerator at 4°C with DI water covering the sediment before eDNA extraction.",
         null,
         "not applicable",
         null,
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "Solution CD2",
         "15000",
         "1",
         "25",
         "4",
         "other: formalin",
         "not applicable",
         "0",
         "not applicable",
         "not applicable",
         "2023-06-27 00:00:00",
         "0.297",
         "g",
         "chemical",
         "magnetic beads",
         "https://www.qiagen.com/us/resources/resourcedetail?id=5a0517a7-711d-4085-8a28-2bb25fab828a&lang=en",
         "Qiagen PowerSoil Kit",
         "https://github.com/aomlomics/protocol_extract_dna_sedtrap_powersoil/blob/main/protocol_extract_dna_sedtrap_powersoil.md",
         "1",
         "NEBNext DNA Repair",
         "4.8015",
         "ng/µl",
         "Quant-iT 1X dsDNA Assay Kit (Invitrogen)",
         "32",
         "uL",
         "153.648",
         "ng",
         "1",
         "Bacteria-16S-V4V5-Parada | Eukarya-18S-V9-Lane-Medlin",
         "550",
         "550",
         "1100",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "1029.94",
         "41.05",
         "110.2",
         "149.85",
         "667.26",
         "5.85",
         null,
         "Luke Thompson | Emily Osborne | Adrianne Wilson | Gabriella Lirio | Sammy Harding",
         "https://orcid.org/0000-0002-3911-1280 | https://orcid.org/0000-0001-9579-5851 | https://orcid.org/0009-0006-1171-6014 | https://orcid.org/0009-0000-5574-1349 | https://orcid.org/0009-0008-8885-6140",
         "luke.thompson@noaa.gov",
         "NOAA Atlantic Oceanographic and Meteorological Laboratories | NOAA Ocean Acidification Program",
         "https://ror.org/042r9xb17 | https://ror.org/02bfn4816",
         "Northern Gulf of Mexico Sediment Trap",
         "noaa-aoml-ngmt",
         "sediment trap time series | water column time series",
         "metabarcoding",
         "After sampling, run ~1 L of 5% bleach through tubing lines, then repeat with ~1 L of deionized water. Rinse the 20-L collection carboy 3 times with ~0.5 L DI water (1.5 L total) after sampling. Before the next sample, rinse 20-L carboy 2 times with ~100 mL of sample water (200 mL total).",
         "1",
         "1",
         "FAIRe_NOAA_checklist_v1.0.2.xlsx",
         "2025-11-12 00:00:00",
         "https://creativecommons.org/publicdomain/zero/1.0/",
         "US Government",
         "No Rights Reserved",
         "https://github.com/aomlomics/sedtrap",
         "1",
         "0",
         "not applicable",
         "Eppendorf Mastercycler Nexus Thermal Cycler",
         "Bacteria-16S-V4V5-Parada",
         "not provided",
         "16S rRNA gene sequencing targeting the V4-V5 region using primers 515F and 926R",
         "Prokaryotes, including Bacteria and Archaea",
         "16S rRNA (SSU prokaryote)",
         "V4-V5",
         "411",
         "GTGYCAGCMGCCGCGGTAA",
         "CCGYCAATTYMTTTRAGTTT",
         "515F-Y",
         "926R",
         "10.1111/1462-2920.13023",
         "10.1111/1462-2920.13023",
         "1",
         "1",
         "10",
         "10",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "AmpliTaq Gold 360 Master Mix, Thermo Fisher Scientific",
         "PCR reactions were run in 25 uL reaction volumes, with 1.0 uL of DNA, 12.5 uL of AmpliTaq Gold, 9.5 uL of water, and 1.0 uL of each primer (10 uM).",
         "not applicable",
         "not applicable",
         "not applicable",
         "25",
         "1",
         "not applicable",
         "https://doi.org/10.1111/1462-2920.13023",
         "initial denaturation:95_3;denaturation:95_0.75;annealing:50_0.75;elongation:72_1.5;final elongation:72_10;30",
         "50",
         "30",
         "not applicable",
         "not applicable",
         "two-step PCR",
         "not provided",
         "15",
         "DreamTaq Hot Start PCR Master Mix (K9012), Thermo Fisher Scientific",
         "not applicable",
         "1",
         "initial denaturation:95_3;denaturation:95_0.25;annealing:60_0.5;elongation:72_1;final elongation:72_3;15",
         "60",
         "11",
         "not applicable",
         "not provided",
         "not applicable",
         "Michigan State University Research Technology Support Facility Genomics Core",
         "ILLUMINA",
         "Illumina MiSeq [OBI_0002003]",
         "MiSeq v2 500",
         "ACACTGACGACATGGTTCTACA",
         "TACGGTAGCAGAGACTTGGTCT",
         "The Genomics Core performed secondary PCR using dual indexed, Illumina-compatible primers targeting the Fluidigm CS1/CS2 oligomers at the ends of the primary PCR products. Amplicons were batch normalized using Invitrogen SequalPrep DNA Normalization plates and recovered product was pooled. The pool was QC’d and quantified using a combination of Qubit dsDNA HS, Agilent 4200 TapeStation HS DNA1000, and Invitrogen Collibri Library Quantification qPCR assays.",
         "MD5",
         "PhiX control library was spiked in at 10%.",
         "32PE | 320G"
        ],
        [
         "192",
         "NGMT_GMT1_Sed_550m_10A_Extracted2306",
         "Bacteria-16S-V4V5-Parada",
         "GMT-2 Sed 16S Redo 2024-01-26",
         "D1",
         "4",
         "GMT1_dup_16S_10A_S58",
         "20240325_16S-Amplicon_PE250",
         "GMT1_dup_16S_10A_S58_L001_R1_001.fastq.gz",
         "GMT1_dup_16S_10A_S58_L001_R2_001.fastq.gz",
         "404331aa25b1ac9374de1c00eac5838f",
         "3ca4d18abec92e9a38c7f388116e22aa",
         "GMT1_10A_dup",
         "sample",
         "not applicable",
         "not applicable",
         "GMT-1",
         "SW01",
         "not applicable",
         "A",
         "re extracted GMT-1 samples (extracted with GMT-2 samples, June 2023)",
         null,
         null,
         "NGMT_GMT1_Sed_550m_10",
         "NGMT_GMT1_Sed_550m_10",
         "not applicable",
         "NGMT_GMT1_Sed_550m_10B_dup | NGMT_GMT1_Sed_550m_10C_dup",
         "GMT-2 Sed DNA",
         "4",
         "D1",
         "2023-06-27 00:00:00",
         "-89.1180000000002",
         "28.2399999999999",
         "89°7.100'W",
         "28°14.387'N",
         "DDM",
         "WGS84",
         "28",
         "-89",
         "USA: Gulf of Mexico",
         "2022-03-29/2022-04-12",
         "2022-03-29",
         "2022-04-05 00:00:00",
         "2022-04-12 00:00:00",
         "P14D",
         "29-Mar-2022/12-Apr-2022",
         "not applicable",
         "407",
         "marginal sea biome (ENVO:01000046)",
         "marine aphotic zone (ENVO:00000210)",
         "marine mud (ENVO:00005795)",
         "0",
         null,
         "sediment trap",
         "0.297",
         "mg",
         "4",
         "other: formalin",
         "P1Y2M15D",
         "AOML Room 248, 4° refridgerator",
         "AOML Room 248, -20° freezer",
         "Sediment samples were stored in a refridgerator at 4°C with DI water covering the sediment before eDNA extraction.",
         null,
         "not applicable",
         null,
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "Solution CD2",
         "15000",
         "1",
         "25",
         "4",
         "other: formalin",
         "not applicable",
         "0",
         "not applicable",
         "not applicable",
         "2023-06-27 00:00:00",
         "0.297",
         "g",
         "chemical",
         "magnetic beads",
         "https://www.qiagen.com/us/resources/resourcedetail?id=5a0517a7-711d-4085-8a28-2bb25fab828a&lang=en",
         "Qiagen PowerSoil Kit",
         "https://github.com/aomlomics/protocol_extract_dna_sedtrap_powersoil/blob/main/protocol_extract_dna_sedtrap_powersoil.md",
         "1",
         "NEBNext DNA Repair",
         "4.8015",
         "ng/µl",
         "Quant-iT 1X dsDNA Assay Kit (Invitrogen)",
         "32",
         "uL",
         "153.648",
         "ng",
         "1",
         "Bacteria-16S-V4V5-Parada | Eukarya-18S-V9-Lane-Medlin",
         "550",
         "550",
         "1100",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "1029.94",
         "41.05",
         "110.2",
         "149.85",
         "667.26",
         "5.85",
         null,
         "Luke Thompson | Emily Osborne | Adrianne Wilson | Gabriella Lirio | Sammy Harding",
         "https://orcid.org/0000-0002-3911-1280 | https://orcid.org/0000-0001-9579-5851 | https://orcid.org/0009-0006-1171-6014 | https://orcid.org/0009-0000-5574-1349 | https://orcid.org/0009-0008-8885-6140",
         "luke.thompson@noaa.gov",
         "NOAA Atlantic Oceanographic and Meteorological Laboratories | NOAA Ocean Acidification Program",
         "https://ror.org/042r9xb17 | https://ror.org/02bfn4816",
         "Northern Gulf of Mexico Sediment Trap",
         "noaa-aoml-ngmt",
         "sediment trap time series | water column time series",
         "metabarcoding",
         "After sampling, run ~1 L of 5% bleach through tubing lines, then repeat with ~1 L of deionized water. Rinse the 20-L collection carboy 3 times with ~0.5 L DI water (1.5 L total) after sampling. Before the next sample, rinse 20-L carboy 2 times with ~100 mL of sample water (200 mL total).",
         "1",
         "1",
         "FAIRe_NOAA_checklist_v1.0.2.xlsx",
         "2025-11-12 00:00:00",
         "https://creativecommons.org/publicdomain/zero/1.0/",
         "US Government",
         "No Rights Reserved",
         "https://github.com/aomlomics/sedtrap",
         "1",
         "0",
         "not applicable",
         "Eppendorf Mastercycler Nexus Thermal Cycler",
         "Bacteria-16S-V4V5-Parada",
         "not provided",
         "16S rRNA gene sequencing targeting the V4-V5 region using primers 515F and 926R",
         "Prokaryotes, including Bacteria and Archaea",
         "16S rRNA (SSU prokaryote)",
         "V4-V5",
         "411",
         "GTGYCAGCMGCCGCGGTAA",
         "CCGYCAATTYMTTTRAGTTT",
         "515F-Y",
         "926R",
         "10.1111/1462-2920.13023",
         "10.1111/1462-2920.13023",
         "1",
         "1",
         "10",
         "10",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "AmpliTaq Gold 360 Master Mix, Thermo Fisher Scientific",
         "PCR reactions were run in 25 uL reaction volumes, with 1.0 uL of DNA, 12.5 uL of AmpliTaq Gold, 9.5 uL of water, and 1.0 uL of each primer (10 uM).",
         "not applicable",
         "not applicable",
         "not applicable",
         "25",
         "1",
         "not applicable",
         "https://doi.org/10.1111/1462-2920.13023",
         "initial denaturation:95_3;denaturation:95_0.75;annealing:50_0.75;elongation:72_1.5;final elongation:72_10;30",
         "50",
         "30",
         "not applicable",
         "not applicable",
         "two-step PCR",
         "not provided",
         "15",
         "DreamTaq Hot Start PCR Master Mix (K9012), Thermo Fisher Scientific",
         "not applicable",
         "1",
         "initial denaturation:95_3;denaturation:95_0.25;annealing:60_0.5;elongation:72_1;final elongation:72_3;15",
         "60",
         "11",
         "not applicable",
         "not provided",
         "not applicable",
         "Michigan State University Research Technology Support Facility Genomics Core",
         "ILLUMINA",
         "Illumina MiSeq [OBI_0002003]",
         "MiSeq v2 500",
         "ACACTGACGACATGGTTCTACA",
         "TACGGTAGCAGAGACTTGGTCT",
         "The Genomics Core performed secondary PCR using dual indexed, Illumina-compatible primers targeting the Fluidigm CS1/CS2 oligomers at the ends of the primary PCR products. Amplicons were batch normalized using Invitrogen SequalPrep DNA Normalization plates and recovered product was pooled. The pool was QC’d and quantified using a combination of Qubit dsDNA HS, Agilent 4200 TapeStation HS DNA1000, and Invitrogen Collibri Library Quantification qPCR assays.",
         "MD5",
         "PhiX control library was spiked in at 10%.",
         "32PE | 320G"
        ],
        [
         "193",
         "NGMT_GMT1_Sed_550m_10A_Extracted2401",
         "Bacteria-16S-V4V5-Parada",
         "GMT-4 Sed + GMT-5 CTD 16S 2024-01-25",
         "C1",
         "3",
         "GMT1_dup3_16S_10A_S3",
         "20240325_16S-Amplicon_PE250",
         "GMT1_dup3_16S_10A_S3_L001_R1_001.fastq.gz",
         "GMT1_dup3_16S_10A_S3_L001_R2_001.fastq.gz",
         "f5481be9c865798c1a07e4f15329d1c8",
         "7c84ee1ef10abaed89491df99e7e05f0",
         "GMT1_10A_dup3",
         "sample",
         "not applicable",
         "not applicable",
         "GMT-1",
         "SW01",
         "not applicable",
         "A",
         "re extracted GMT-1 samples (extracted with GMT-4 samples, January 2024)",
         null,
         null,
         "NGMT_GMT1_Sed_550m_10",
         "NGMT_GMT1_Sed_550m_10",
         "not applicable",
         "NGMT_GMT1_Sed_550m_10B_dup3 | NGMT_GMT1_Sed_550m_10C_dup3 | NGMT_GMT1_Sed_550m_10D_dup3",
         "GMT-4 Sed DNA",
         "3",
         "C1",
         "2024-01-23 00:00:00",
         "-89.1180000000001",
         "28.2399999999999",
         "89°7.100'W",
         "28°14.387'N",
         "DDM",
         "WGS84",
         "28",
         "-89",
         "USA: Gulf of Mexico",
         "2022-03-29/2022-04-12",
         "2022-03-29",
         "2022-04-05 00:00:00",
         "2022-04-12 00:00:00",
         "P14D",
         "29-Mar-2022/12-Apr-2022",
         "not applicable",
         "617",
         "marginal sea biome (ENVO:01000046)",
         "marine aphotic zone (ENVO:00000210)",
         "marine mud (ENVO:00005795)",
         "0",
         null,
         "sediment trap",
         "0.256",
         "mg",
         "4",
         "other: formalin",
         "P1Y9M11D",
         "AOML Room 248, 4° refridgerator",
         "AOML Room 248, -20° freezer",
         "Sediment samples were stored in a refridgerator at 4°C with DI water covering the sediment before eDNA extraction.",
         null,
         "not applicable",
         null,
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "Solution CD2",
         "15000",
         "1",
         "25",
         "4",
         "other: formalin",
         "not applicable",
         "0",
         "not applicable",
         "not applicable",
         "2024-01-23 00:00:00",
         "0.256",
         "g",
         "chemical",
         "magnetic beads",
         "https://www.qiagen.com/us/resources/resourcedetail?id=5a0517a7-711d-4085-8a28-2bb25fab828a&lang=en",
         "Qiagen PowerSoil Kit",
         "https://github.com/aomlomics/protocol_extract_dna_sedtrap_powersoil/blob/main/protocol_extract_dna_sedtrap_powersoil.md",
         "1",
         "NEBNext DNA Repair",
         "18.72",
         "ng/µl",
         "Quant-iT 1X dsDNA Assay Kit (Invitrogen)",
         "32",
         "uL",
         "599.04",
         "ng",
         "1",
         "Bacteria-16S-V4V5-Parada | Eukarya-18S-V9-Lane-Medlin",
         "550",
         "550",
         "1100",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "1029.94",
         "41.05",
         "110.2",
         "149.85",
         "667.26",
         "5.85",
         null,
         "Luke Thompson | Emily Osborne | Adrianne Wilson | Gabriella Lirio | Sammy Harding",
         "https://orcid.org/0000-0002-3911-1280 | https://orcid.org/0000-0001-9579-5851 | https://orcid.org/0009-0006-1171-6014 | https://orcid.org/0009-0000-5574-1349 | https://orcid.org/0009-0008-8885-6140",
         "luke.thompson@noaa.gov",
         "NOAA Atlantic Oceanographic and Meteorological Laboratories | NOAA Ocean Acidification Program",
         "https://ror.org/042r9xb17 | https://ror.org/02bfn4816",
         "Northern Gulf of Mexico Sediment Trap",
         "noaa-aoml-ngmt",
         "sediment trap time series | water column time series",
         "metabarcoding",
         "After sampling, run ~1 L of 5% bleach through tubing lines, then repeat with ~1 L of deionized water. Rinse the 20-L collection carboy 3 times with ~0.5 L DI water (1.5 L total) after sampling. Before the next sample, rinse 20-L carboy 2 times with ~100 mL of sample water (200 mL total).",
         "1",
         "1",
         "FAIRe_NOAA_checklist_v1.0.2.xlsx",
         "2025-11-12 00:00:00",
         "https://creativecommons.org/publicdomain/zero/1.0/",
         "US Government",
         "No Rights Reserved",
         "https://github.com/aomlomics/sedtrap",
         "1",
         "0",
         "not applicable",
         "Eppendorf Mastercycler Nexus Thermal Cycler",
         "Bacteria-16S-V4V5-Parada",
         "not provided",
         "16S rRNA gene sequencing targeting the V4-V5 region using primers 515F and 926R",
         "Prokaryotes, including Bacteria and Archaea",
         "16S rRNA (SSU prokaryote)",
         "V4-V5",
         "411",
         "GTGYCAGCMGCCGCGGTAA",
         "CCGYCAATTYMTTTRAGTTT",
         "515F-Y",
         "926R",
         "10.1111/1462-2920.13023",
         "10.1111/1462-2920.13023",
         "1",
         "1",
         "10",
         "10",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "AmpliTaq Gold 360 Master Mix, Thermo Fisher Scientific",
         "PCR reactions were run in 25 uL reaction volumes, with 1.0 uL of DNA, 12.5 uL of AmpliTaq Gold, 9.5 uL of water, and 1.0 uL of each primer (10 uM).",
         "not applicable",
         "not applicable",
         "not applicable",
         "25",
         "1",
         "not applicable",
         "https://doi.org/10.1111/1462-2920.13023",
         "initial denaturation:95_3;denaturation:95_0.75;annealing:50_0.75;elongation:72_1.5;final elongation:72_10;30",
         "50",
         "30",
         "not applicable",
         "not applicable",
         "two-step PCR",
         "not provided",
         "15",
         "DreamTaq Hot Start PCR Master Mix (K9012), Thermo Fisher Scientific",
         "not applicable",
         "1",
         "initial denaturation:95_3;denaturation:95_0.25;annealing:60_0.5;elongation:72_1;final elongation:72_3;15",
         "60",
         "11",
         "not applicable",
         "not provided",
         "not applicable",
         "Michigan State University Research Technology Support Facility Genomics Core",
         "ILLUMINA",
         "Illumina MiSeq [OBI_0002003]",
         "MiSeq v2 500",
         "ACACTGACGACATGGTTCTACA",
         "TACGGTAGCAGAGACTTGGTCT",
         "The Genomics Core performed secondary PCR using dual indexed, Illumina-compatible primers targeting the Fluidigm CS1/CS2 oligomers at the ends of the primary PCR products. Amplicons were batch normalized using Invitrogen SequalPrep DNA Normalization plates and recovered product was pooled. The pool was QC’d and quantified using a combination of Qubit dsDNA HS, Agilent 4200 TapeStation HS DNA1000, and Invitrogen Collibri Library Quantification qPCR assays.",
         "MD5",
         "PhiX control library was spiked in at 10%.",
         "32PE | 320G"
        ],
        [
         "291",
         "NGMT_GMT1_Sed_550m_10A_Extracted2412",
         "Bacteria-16S-V4V5-Parada",
         "GMT-5 Sed 16S 2024-12-18",
         "H8",
         "64",
         "GMT1_10A_dup4_16S_S64",
         "20250423_16S-Amplicon_PE250",
         "GMT1_10A_dup4_16S_S64_L001_R1_001.fastq.gz",
         "GMT1_10A_dup4_16S_S64_L001_R2_001.fastq.gz",
         "58e9115f5c418a301623e380546194b3",
         "cfba63dc93dde9da7bab60d0f799c3aa",
         "GMT1_10A_dup4",
         "sample",
         "not applicable",
         "not applicable",
         "GMT-1",
         "SW01",
         "not applicable",
         "A",
         "sample opened in bead vortexing- lost bit of sample",
         null,
         null,
         "NGMT_GMT1_Sed_550m_10",
         "NGMT_GMT1_Sed_550m_10",
         "not applicable",
         "NGMT_GMT1_Sed_550m_10B_dup4 | NGMT_GMT1_Sed_550m_10C_dup4",
         "GMT-5 Sed DNA",
         "64",
         "H8",
         "2024-12-12 00:00:00",
         "-89.1180000000002",
         "28.2399999999999",
         "89°7.100'W",
         "28°14.387'N",
         "DDM",
         "WGS84",
         "28",
         "-89",
         "USA: Gulf of Mexico",
         "2022-03-29/2022-04-12",
         "2022-03-29",
         "2022-04-05 00:00:00",
         "2022-04-12 00:00:00",
         "P14D",
         "29-Mar-2022/12-Apr-2022",
         "not applicable",
         "941",
         "marginal sea biome (ENVO:01000046)",
         "marine aphotic zone (ENVO:00000210)",
         "marine mud (ENVO:00005795)",
         "0",
         null,
         "sediment trap",
         "0.249",
         "mg",
         "4",
         "other: formalin",
         "P2Y8M",
         "AOML Room 248, 4° refridgerator",
         "AOML Room 248, -20° freezer",
         "Sediment samples were stored in a refridgerator at 4°C with DI water covering the sediment before eDNA extraction.",
         null,
         "not applicable",
         null,
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "Solution CD2",
         "15000",
         "1",
         "25",
         "4",
         "other: formalin",
         "not applicable",
         "0",
         "not applicable",
         "not applicable",
         "2024-12-12 00:00:00",
         ".249",
         "g",
         "chemical",
         "magnetic beads",
         "https://www.qiagen.com/us/resources/resourcedetail?id=5a0517a7-711d-4085-8a28-2bb25fab828a&lang=en",
         "Qiagen PowerSoil Kit",
         "https://github.com/aomlomics/protocol_extract_dna_sedtrap_powersoil/blob/main/protocol_extract_dna_sedtrap_powersoil.md",
         "1",
         "NEBNext DNA Repair",
         "3.658",
         "ng/µl",
         "Quant-iT 1X dsDNA Assay Kit (Invitrogen)",
         "32",
         "uL",
         "117.056",
         "ng",
         "1",
         "Bacteria-16S-V4V5-Parada | Eukarya-18S-V9-Lane-Medlin",
         "550",
         "550",
         "1100",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "1029.94",
         "41.05",
         "110.2",
         "149.85",
         "667.26",
         "5.85",
         null,
         "Luke Thompson | Emily Osborne | Adrianne Wilson | Gabriella Lirio | Sammy Harding",
         "https://orcid.org/0000-0002-3911-1280 | https://orcid.org/0000-0001-9579-5851 | https://orcid.org/0009-0006-1171-6014 | https://orcid.org/0009-0000-5574-1349 | https://orcid.org/0009-0008-8885-6140",
         "luke.thompson@noaa.gov",
         "NOAA Atlantic Oceanographic and Meteorological Laboratories | NOAA Ocean Acidification Program",
         "https://ror.org/042r9xb17 | https://ror.org/02bfn4816",
         "Northern Gulf of Mexico Sediment Trap",
         "noaa-aoml-ngmt",
         "sediment trap time series | water column time series",
         "metabarcoding",
         "After sampling, run ~1 L of 5% bleach through tubing lines, then repeat with ~1 L of deionized water. Rinse the 20-L collection carboy 3 times with ~0.5 L DI water (1.5 L total) after sampling. Before the next sample, rinse 20-L carboy 2 times with ~100 mL of sample water (200 mL total).",
         "1",
         "1",
         "FAIRe_NOAA_checklist_v1.0.2.xlsx",
         "2025-11-12 00:00:00",
         "https://creativecommons.org/publicdomain/zero/1.0/",
         "US Government",
         "No Rights Reserved",
         "https://github.com/aomlomics/sedtrap",
         "1",
         "0",
         "not applicable",
         "Eppendorf Mastercycler Nexus Thermal Cycler",
         "Bacteria-16S-V4V5-Parada",
         "not provided",
         "16S rRNA gene sequencing targeting the V4-V5 region using primers 515F and 926R",
         "Prokaryotes, including Bacteria and Archaea",
         "16S rRNA (SSU prokaryote)",
         "V4-V5",
         "411",
         "GTGYCAGCMGCCGCGGTAA",
         "CCGYCAATTYMTTTRAGTTT",
         "515F-Y",
         "926R",
         "10.1111/1462-2920.13023",
         "10.1111/1462-2920.13023",
         "1",
         "1",
         "10",
         "10",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "AmpliTaq Gold 360 Master Mix, Thermo Fisher Scientific",
         "PCR reactions were run in 25 uL reaction volumes, with 1.0 uL of DNA, 12.5 uL of AmpliTaq Gold, 9.5 uL of water, and 1.0 uL of each primer (10 uM).",
         "not applicable",
         "not applicable",
         "not applicable",
         "25",
         "1",
         "not applicable",
         "https://doi.org/10.1111/1462-2920.13023",
         "initial denaturation:95_3;denaturation:95_0.75;annealing:50_0.75;elongation:72_1.5;final elongation:72_10;30",
         "50",
         "30",
         "not applicable",
         "not applicable",
         "two-step PCR",
         "not provided",
         "15",
         "DreamTaq Hot Start PCR Master Mix (K9012), Thermo Fisher Scientific",
         "not applicable",
         "1",
         "initial denaturation:95_3;denaturation:95_0.25;annealing:60_0.5;elongation:72_1;final elongation:72_3;15",
         "60",
         "11",
         "not applicable",
         "not provided",
         "not applicable",
         "Michigan State University Research Technology Support Facility Genomics Core",
         "ILLUMINA",
         "Illumina MiSeq [OBI_0002003]",
         "MiSeq v2 500",
         "ACACTGACGACATGGTTCTACA",
         "TACGGTAGCAGAGACTTGGTCT",
         "The Genomics Core performed secondary PCR using dual indexed, Illumina-compatible primers targeting the Fluidigm CS1/CS2 oligomers at the ends of the primary PCR products. Amplicons were batch normalized using Invitrogen SequalPrep DNA Normalization plates and recovered product was pooled. The pool was QC’d and quantified using a combination of Qubit dsDNA HS, Agilent 4200 TapeStation HS DNA1000, and Invitrogen Collibri Library Quantification qPCR assays.",
         "MD5",
         "PhiX control library was spiked in at 10%.",
         "32PE | 320G"
        ],
        [
         "380",
         "NGMT_GMT1_Sed_550m_10A_Extracted2306",
         "Bacteria-16S-V4V5-Parada",
         "GMT-2 Sed 16S Redo 2025-06-26",
         "D1",
         "4",
         "GMT1_10A_dup_16S_S65",
         "20250728_THO16883_16S-Amplicon_PE250",
         "GMT1_10A_dup_16S_S65_L001_R1_001.fastq.gz",
         "GMT1_10A_dup_16S_S65_L001_R2_001.fastq.gz",
         "2852e9d675100a17471bd0d04dcbd07c",
         "6a34379d3b1ac7719ef39736b26ad261",
         "GMT1_10A_dup",
         "sample",
         "not applicable",
         "not applicable",
         "GMT-1",
         "SW01",
         "not applicable",
         "A",
         "re extracted GMT-1 samples (extracted with GMT-2 samples, June 2023)",
         null,
         null,
         "NGMT_GMT1_Sed_550m_10",
         "NGMT_GMT1_Sed_550m_10",
         "not applicable",
         "NGMT_GMT1_Sed_550m_10B_dup | NGMT_GMT1_Sed_550m_10C_dup",
         "GMT-2 Sed DNA",
         "4",
         "D1",
         "2023-06-27 00:00:00",
         "-89.1180000000002",
         "28.2399999999999",
         "89°7.100'W",
         "28°14.387'N",
         "DDM",
         "WGS84",
         "28",
         "-89",
         "USA: Gulf of Mexico",
         "2022-03-29/2022-04-12",
         "2022-03-29",
         "2022-04-05 00:00:00",
         "2022-04-12 00:00:00",
         "P14D",
         "29-Mar-2022/12-Apr-2022",
         "not applicable",
         "407",
         "marginal sea biome (ENVO:01000046)",
         "marine aphotic zone (ENVO:00000210)",
         "marine mud (ENVO:00005795)",
         "0",
         null,
         "sediment trap",
         "0.297",
         "mg",
         "4",
         "other: formalin",
         "P1Y2M15D",
         "AOML Room 248, 4° refridgerator",
         "AOML Room 248, -20° freezer",
         "Sediment samples were stored in a refridgerator at 4°C with DI water covering the sediment before eDNA extraction.",
         null,
         "not applicable",
         null,
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "Solution CD2",
         "15000",
         "1",
         "25",
         "4",
         "other: formalin",
         "not applicable",
         "0",
         "not applicable",
         "not applicable",
         "2023-06-27 00:00:00",
         "0.297",
         "g",
         "chemical",
         "magnetic beads",
         "https://www.qiagen.com/us/resources/resourcedetail?id=5a0517a7-711d-4085-8a28-2bb25fab828a&lang=en",
         "Qiagen PowerSoil Kit",
         "https://github.com/aomlomics/protocol_extract_dna_sedtrap_powersoil/blob/main/protocol_extract_dna_sedtrap_powersoil.md",
         "1",
         "NEBNext DNA Repair",
         "4.8015",
         "ng/µl",
         "Quant-iT 1X dsDNA Assay Kit (Invitrogen)",
         "32",
         "uL",
         "153.648",
         "ng",
         "1",
         "Bacteria-16S-V4V5-Parada | Eukarya-18S-V9-Lane-Medlin",
         "550",
         "550",
         "1100",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "1029.94",
         "41.05",
         "110.2",
         "149.85",
         "667.26",
         "5.85",
         null,
         "Luke Thompson | Emily Osborne | Adrianne Wilson | Gabriella Lirio | Sammy Harding",
         "https://orcid.org/0000-0002-3911-1280 | https://orcid.org/0000-0001-9579-5851 | https://orcid.org/0009-0006-1171-6014 | https://orcid.org/0009-0000-5574-1349 | https://orcid.org/0009-0008-8885-6140",
         "luke.thompson@noaa.gov",
         "NOAA Atlantic Oceanographic and Meteorological Laboratories | NOAA Ocean Acidification Program",
         "https://ror.org/042r9xb17 | https://ror.org/02bfn4816",
         "Northern Gulf of Mexico Sediment Trap",
         "noaa-aoml-ngmt",
         "sediment trap time series | water column time series",
         "metabarcoding",
         "After sampling, run ~1 L of 5% bleach through tubing lines, then repeat with ~1 L of deionized water. Rinse the 20-L collection carboy 3 times with ~0.5 L DI water (1.5 L total) after sampling. Before the next sample, rinse 20-L carboy 2 times with ~100 mL of sample water (200 mL total).",
         "1",
         "1",
         "FAIRe_NOAA_checklist_v1.0.2.xlsx",
         "2025-11-12 00:00:00",
         "https://creativecommons.org/publicdomain/zero/1.0/",
         "US Government",
         "No Rights Reserved",
         "https://github.com/aomlomics/sedtrap",
         "1",
         "0",
         "not applicable",
         "Eppendorf Mastercycler Nexus Thermal Cycler",
         "Bacteria-16S-V4V5-Parada",
         "not provided",
         "16S rRNA gene sequencing targeting the V4-V5 region using primers 515F and 926R",
         "Prokaryotes, including Bacteria and Archaea",
         "16S rRNA (SSU prokaryote)",
         "V4-V5",
         "411",
         "GTGYCAGCMGCCGCGGTAA",
         "CCGYCAATTYMTTTRAGTTT",
         "515F-Y",
         "926R",
         "10.1111/1462-2920.13023",
         "10.1111/1462-2920.13023",
         "1",
         "1",
         "10",
         "10",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "AmpliTaq Gold 360 Master Mix, Thermo Fisher Scientific",
         "PCR reactions were run in 25 uL reaction volumes, with 1.0 uL of DNA, 12.5 uL of AmpliTaq Gold, 9.5 uL of water, and 1.0 uL of each primer (10 uM).",
         "not applicable",
         "not applicable",
         "not applicable",
         "25",
         "1",
         "not applicable",
         "https://doi.org/10.1111/1462-2920.13023",
         "initial denaturation:95_3;denaturation:95_0.75;annealing:50_0.75;elongation:72_1.5;final elongation:72_10;30",
         "50",
         "30",
         "not applicable",
         "not applicable",
         "two-step PCR",
         "not provided",
         "15",
         "DreamTaq Hot Start PCR Master Mix (K9012), Thermo Fisher Scientific",
         "not applicable",
         "1",
         "initial denaturation:95_3;denaturation:95_0.25;annealing:60_0.5;elongation:72_1;final elongation:72_3;15",
         "60",
         "11",
         "not applicable",
         "not provided",
         "not applicable",
         "Michigan State University Research Technology Support Facility Genomics Core",
         "ILLUMINA",
         "Illumina MiSeq [OBI_0002003]",
         "MiSeq v2 500",
         "ACACTGACGACATGGTTCTACA",
         "TACGGTAGCAGAGACTTGGTCT",
         "The Genomics Core performed secondary PCR using dual indexed, Illumina-compatible primers targeting the Fluidigm CS1/CS2 oligomers at the ends of the primary PCR products. Amplicons were batch normalized using Invitrogen SequalPrep DNA Normalization plates and recovered product was pooled. The pool was QC’d and quantified using a combination of Qubit dsDNA HS, Agilent 4200 TapeStation HS DNA1000, and Invitrogen Collibri Library Quantification qPCR assays.",
         "MD5",
         "PhiX control library was spiked in at 10%.",
         "32PE | 320G"
        ],
        [
         "381",
         "NGMT_GMT1_Sed_550m_10A_Extracted2506",
         "Bacteria-16S-V4V5-Parada",
         "GMT-6 Sed + GMT-7 CTD 16S 2025-06-26",
         "D7",
         "52",
         "GMT1_10A_dup5_16S_S52",
         "20250728_THO16883_16S-Amplicon_PE250",
         "GMT1_10A_dup5_16S_S52_L001_R1_001.fastq.gz",
         "GMT1_10A_dup5_16S_S52_L001_R2_001.fastq.gz",
         "41e619ead2a35132ee2309cc443391b3",
         "ed29de98bea41f5300d7cf193145ab7b",
         "GMT1_10A_dup5",
         "sample",
         "not applicable",
         "not applicable",
         "GMT-1",
         "SW01",
         "not applicable",
         "A",
         "re-extracting sample",
         null,
         null,
         "NGMT_GMT1_Sed_550m_10",
         "NGMT_GMT1_Sed_550m_10",
         "not applicable",
         "NGMT_GMT1_Sed_550m_10B_dup5 | NGMT_GMT1_Sed_550m_10C_dup5",
         "GMT-6 repaired DNA",
         "52",
         "D7",
         "2025-06-16 00:00:00",
         "-89.1180000000002",
         "28.2399999999999",
         "89°7.100'W",
         "28°14.387'N",
         "DDM",
         "WGS84",
         "28",
         "-89",
         "USA: Gulf of Mexico",
         "2022-03-29/2022-04-12",
         "2022-03-29",
         "2022-04-05 00:00:00",
         "2022-04-12 00:00:00",
         "P14D",
         "29-Mar-2022/12-Apr-2022",
         "not applicable",
         "1127",
         "marginal sea biome (ENVO:01000046)",
         "marine aphotic zone (ENVO:00000210)",
         "marine mud (ENVO:00005795)",
         "0",
         null,
         "sediment trap",
         "0.26",
         "mg",
         "4",
         "other: formalin",
         "P3Y2M4D",
         "AOML Room 248, 4° refridgerator",
         "AOML Room 248, -20° freezer",
         "Sediment samples were stored in a refridgerator at 4°C with DI water covering the sediment before eDNA extraction.",
         null,
         "not applicable",
         null,
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "Solution CD2",
         "15000",
         "1",
         "25",
         "4",
         "other: formalin",
         "not applicable",
         "0",
         "not applicable",
         "not applicable",
         "2025-06-16 00:00:00",
         "0.26",
         "g",
         "chemical",
         "magnetic beads",
         "https://www.qiagen.com/us/resources/resourcedetail?id=5a0517a7-711d-4085-8a28-2bb25fab828a&lang=en",
         "Qiagen PowerSoil Kit",
         "https://github.com/aomlomics/protocol_extract_dna_sedtrap_powersoil/blob/main/protocol_extract_dna_sedtrap_powersoil.md",
         "1",
         "NEBNext DNA Repair",
         "missing: not collected",
         "missing: not collected",
         "missing: not collected",
         "32",
         "uL",
         "missing: not collected",
         "missing: not collected",
         "missing: not collected",
         "Bacteria-16S-V4V5-Parada | Eukarya-18S-V9-Lane-Medlin",
         "550",
         "550",
         "1100",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "1029.94",
         "41.05",
         "110.2",
         "149.85",
         "667.26",
         "5.85",
         null,
         "Luke Thompson | Emily Osborne | Adrianne Wilson | Gabriella Lirio | Sammy Harding",
         "https://orcid.org/0000-0002-3911-1280 | https://orcid.org/0000-0001-9579-5851 | https://orcid.org/0009-0006-1171-6014 | https://orcid.org/0009-0000-5574-1349 | https://orcid.org/0009-0008-8885-6140",
         "luke.thompson@noaa.gov",
         "NOAA Atlantic Oceanographic and Meteorological Laboratories | NOAA Ocean Acidification Program",
         "https://ror.org/042r9xb17 | https://ror.org/02bfn4816",
         "Northern Gulf of Mexico Sediment Trap",
         "noaa-aoml-ngmt",
         "sediment trap time series | water column time series",
         "metabarcoding",
         "After sampling, run ~1 L of 5% bleach through tubing lines, then repeat with ~1 L of deionized water. Rinse the 20-L collection carboy 3 times with ~0.5 L DI water (1.5 L total) after sampling. Before the next sample, rinse 20-L carboy 2 times with ~100 mL of sample water (200 mL total).",
         "1",
         "1",
         "FAIRe_NOAA_checklist_v1.0.2.xlsx",
         "2025-11-12 00:00:00",
         "https://creativecommons.org/publicdomain/zero/1.0/",
         "US Government",
         "No Rights Reserved",
         "https://github.com/aomlomics/sedtrap",
         "1",
         "0",
         "not applicable",
         "Eppendorf Mastercycler Nexus Thermal Cycler",
         "Bacteria-16S-V4V5-Parada",
         "not provided",
         "16S rRNA gene sequencing targeting the V4-V5 region using primers 515F and 926R",
         "Prokaryotes, including Bacteria and Archaea",
         "16S rRNA (SSU prokaryote)",
         "V4-V5",
         "411",
         "GTGYCAGCMGCCGCGGTAA",
         "CCGYCAATTYMTTTRAGTTT",
         "515F-Y",
         "926R",
         "10.1111/1462-2920.13023",
         "10.1111/1462-2920.13023",
         "1",
         "1",
         "10",
         "10",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "not applicable",
         "AmpliTaq Gold 360 Master Mix, Thermo Fisher Scientific",
         "PCR reactions were run in 25 uL reaction volumes, with 1.0 uL of DNA, 12.5 uL of AmpliTaq Gold, 9.5 uL of water, and 1.0 uL of each primer (10 uM).",
         "not applicable",
         "not applicable",
         "not applicable",
         "25",
         "1",
         "not applicable",
         "https://doi.org/10.1111/1462-2920.13023",
         "initial denaturation:95_3;denaturation:95_0.75;annealing:50_0.75;elongation:72_1.5;final elongation:72_10;30",
         "50",
         "30",
         "not applicable",
         "not applicable",
         "two-step PCR",
         "not provided",
         "15",
         "DreamTaq Hot Start PCR Master Mix (K9012), Thermo Fisher Scientific",
         "not applicable",
         "1",
         "initial denaturation:95_3;denaturation:95_0.25;annealing:60_0.5;elongation:72_1;final elongation:72_3;15",
         "60",
         "11",
         "not applicable",
         "not provided",
         "not applicable",
         "Michigan State University Research Technology Support Facility Genomics Core",
         "ILLUMINA",
         "Illumina MiSeq [OBI_0002003]",
         "MiSeq v2 500",
         "ACACTGACGACATGGTTCTACA",
         "TACGGTAGCAGAGACTTGGTCT",
         "The Genomics Core performed secondary PCR using dual indexed, Illumina-compatible primers targeting the Fluidigm CS1/CS2 oligomers at the ends of the primary PCR products. Amplicons were batch normalized using Invitrogen SequalPrep DNA Normalization plates and recovered product was pooled. The pool was QC’d and quantified using a combination of Qubit dsDNA HS, Agilent 4200 TapeStation HS DNA1000, and Invitrogen Collibri Library Quantification qPCR assays.",
         "MD5",
         "PhiX control library was spiked in at 10%.",
         "32PE | 320G"
        ]
       ],
       "shape": {
        "columns": 205,
        "rows": 8
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>sample_name</th>\n",
       "      <th>assay_name</th>\n",
       "      <th>pcr_plate_id</th>\n",
       "      <th>pcr_well_position</th>\n",
       "      <th>pcr_well_number</th>\n",
       "      <th>lib_id</th>\n",
       "      <th>seq_run_id</th>\n",
       "      <th>filename</th>\n",
       "      <th>filename2</th>\n",
       "      <th>checksum_filename</th>\n",
       "      <th>...</th>\n",
       "      <th>sequencing_location</th>\n",
       "      <th>platform</th>\n",
       "      <th>instrument</th>\n",
       "      <th>seq_kit</th>\n",
       "      <th>adapter_forward</th>\n",
       "      <th>adapter_reverse</th>\n",
       "      <th>lib_screen</th>\n",
       "      <th>checksum_method</th>\n",
       "      <th>seq_method_additional</th>\n",
       "      <th>ship_crs_expocode</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>NGMT_GMT1_Sed_550m_10A</td>\n",
       "      <td>Bacteria-16S-V4V5-Parada</td>\n",
       "      <td>GMT-1 Sed 16S 2022-09</td>\n",
       "      <td>D4</td>\n",
       "      <td>28</td>\n",
       "      <td>GMT1_16S_10A_S28</td>\n",
       "      <td>20220912_16S-Amplicon_PE250</td>\n",
       "      <td>GMT1_16S_10A_S28_L001_R1_001.fastq.gz</td>\n",
       "      <td>GMT1_16S_10A_S28_L001_R2_001.fastq.gz</td>\n",
       "      <td>d678bd9054c9e861f2e21f3b19f050d7</td>\n",
       "      <td>...</td>\n",
       "      <td>Michigan State University Research Technology ...</td>\n",
       "      <td>ILLUMINA</td>\n",
       "      <td>Illumina MiSeq [OBI_0002003]</td>\n",
       "      <td>MiSeq v2 500</td>\n",
       "      <td>ACACTGACGACATGGTTCTACA</td>\n",
       "      <td>TACGGTAGCAGAGACTTGGTCT</td>\n",
       "      <td>The Genomics Core performed secondary PCR usin...</td>\n",
       "      <td>MD5</td>\n",
       "      <td>PhiX control library was spiked in at 10%.</td>\n",
       "      <td>32PE | 320G</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>NGMT_GMT1_Sed_550m_10A_Extracted2306</td>\n",
       "      <td>Bacteria-16S-V4V5-Parada</td>\n",
       "      <td>GMT-2 Sed 16S 2023-08</td>\n",
       "      <td>D1</td>\n",
       "      <td>4</td>\n",
       "      <td>GMT1_dup_16S_10A_S4</td>\n",
       "      <td>20231110_AND14513_Amplicon_PE250</td>\n",
       "      <td>GMT1_dup_16S_10A_S4_L001_R1_001.fastq.gz</td>\n",
       "      <td>GMT1_dup_16S_10A_S4_L001_R2_001.fastq.gz</td>\n",
       "      <td>3c544b57b6949fd3ca628f2596c03dc9</td>\n",
       "      <td>...</td>\n",
       "      <td>Michigan State University Research Technology ...</td>\n",
       "      <td>ILLUMINA</td>\n",
       "      <td>Illumina MiSeq [OBI_0002003]</td>\n",
       "      <td>MiSeq v2 500</td>\n",
       "      <td>ACACTGACGACATGGTTCTACA</td>\n",
       "      <td>TACGGTAGCAGAGACTTGGTCT</td>\n",
       "      <td>The Genomics Core performed secondary PCR usin...</td>\n",
       "      <td>MD5</td>\n",
       "      <td>PhiX control library was spiked in at 10%.</td>\n",
       "      <td>32PE | 320G</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50</th>\n",
       "      <td>NGMT_GMT1_Sed_550m_10A_Extracted2306</td>\n",
       "      <td>Bacteria-16S-V4V5-Parada</td>\n",
       "      <td>GMT-3 Sed 16S 2023-07-28</td>\n",
       "      <td>F6</td>\n",
       "      <td>46</td>\n",
       "      <td>GMT1_dup2_16S_10A_S91</td>\n",
       "      <td>20231110_AND14513_Amplicon_PE250</td>\n",
       "      <td>GMT1_dup2_16S_10A_S91_L001_R1_001.fastq.gz</td>\n",
       "      <td>GMT1_dup2_16S_10A_S91_L001_R2_001.fastq.gz</td>\n",
       "      <td>4c762a1dafa1b6094495bf1c9ebcc76f</td>\n",
       "      <td>...</td>\n",
       "      <td>Michigan State University Research Technology ...</td>\n",
       "      <td>ILLUMINA</td>\n",
       "      <td>Illumina MiSeq [OBI_0002003]</td>\n",
       "      <td>MiSeq v2 500</td>\n",
       "      <td>ACACTGACGACATGGTTCTACA</td>\n",
       "      <td>TACGGTAGCAGAGACTTGGTCT</td>\n",
       "      <td>The Genomics Core performed secondary PCR usin...</td>\n",
       "      <td>MD5</td>\n",
       "      <td>PhiX control library was spiked in at 10%.</td>\n",
       "      <td>32PE | 320G</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>192</th>\n",
       "      <td>NGMT_GMT1_Sed_550m_10A_Extracted2306</td>\n",
       "      <td>Bacteria-16S-V4V5-Parada</td>\n",
       "      <td>GMT-2 Sed 16S Redo 2024-01-26</td>\n",
       "      <td>D1</td>\n",
       "      <td>4</td>\n",
       "      <td>GMT1_dup_16S_10A_S58</td>\n",
       "      <td>20240325_16S-Amplicon_PE250</td>\n",
       "      <td>GMT1_dup_16S_10A_S58_L001_R1_001.fastq.gz</td>\n",
       "      <td>GMT1_dup_16S_10A_S58_L001_R2_001.fastq.gz</td>\n",
       "      <td>404331aa25b1ac9374de1c00eac5838f</td>\n",
       "      <td>...</td>\n",
       "      <td>Michigan State University Research Technology ...</td>\n",
       "      <td>ILLUMINA</td>\n",
       "      <td>Illumina MiSeq [OBI_0002003]</td>\n",
       "      <td>MiSeq v2 500</td>\n",
       "      <td>ACACTGACGACATGGTTCTACA</td>\n",
       "      <td>TACGGTAGCAGAGACTTGGTCT</td>\n",
       "      <td>The Genomics Core performed secondary PCR usin...</td>\n",
       "      <td>MD5</td>\n",
       "      <td>PhiX control library was spiked in at 10%.</td>\n",
       "      <td>32PE | 320G</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>193</th>\n",
       "      <td>NGMT_GMT1_Sed_550m_10A_Extracted2401</td>\n",
       "      <td>Bacteria-16S-V4V5-Parada</td>\n",
       "      <td>GMT-4 Sed + GMT-5 CTD 16S 2024-01-25</td>\n",
       "      <td>C1</td>\n",
       "      <td>3</td>\n",
       "      <td>GMT1_dup3_16S_10A_S3</td>\n",
       "      <td>20240325_16S-Amplicon_PE250</td>\n",
       "      <td>GMT1_dup3_16S_10A_S3_L001_R1_001.fastq.gz</td>\n",
       "      <td>GMT1_dup3_16S_10A_S3_L001_R2_001.fastq.gz</td>\n",
       "      <td>f5481be9c865798c1a07e4f15329d1c8</td>\n",
       "      <td>...</td>\n",
       "      <td>Michigan State University Research Technology ...</td>\n",
       "      <td>ILLUMINA</td>\n",
       "      <td>Illumina MiSeq [OBI_0002003]</td>\n",
       "      <td>MiSeq v2 500</td>\n",
       "      <td>ACACTGACGACATGGTTCTACA</td>\n",
       "      <td>TACGGTAGCAGAGACTTGGTCT</td>\n",
       "      <td>The Genomics Core performed secondary PCR usin...</td>\n",
       "      <td>MD5</td>\n",
       "      <td>PhiX control library was spiked in at 10%.</td>\n",
       "      <td>32PE | 320G</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>291</th>\n",
       "      <td>NGMT_GMT1_Sed_550m_10A_Extracted2412</td>\n",
       "      <td>Bacteria-16S-V4V5-Parada</td>\n",
       "      <td>GMT-5 Sed 16S 2024-12-18</td>\n",
       "      <td>H8</td>\n",
       "      <td>64</td>\n",
       "      <td>GMT1_10A_dup4_16S_S64</td>\n",
       "      <td>20250423_16S-Amplicon_PE250</td>\n",
       "      <td>GMT1_10A_dup4_16S_S64_L001_R1_001.fastq.gz</td>\n",
       "      <td>GMT1_10A_dup4_16S_S64_L001_R2_001.fastq.gz</td>\n",
       "      <td>58e9115f5c418a301623e380546194b3</td>\n",
       "      <td>...</td>\n",
       "      <td>Michigan State University Research Technology ...</td>\n",
       "      <td>ILLUMINA</td>\n",
       "      <td>Illumina MiSeq [OBI_0002003]</td>\n",
       "      <td>MiSeq v2 500</td>\n",
       "      <td>ACACTGACGACATGGTTCTACA</td>\n",
       "      <td>TACGGTAGCAGAGACTTGGTCT</td>\n",
       "      <td>The Genomics Core performed secondary PCR usin...</td>\n",
       "      <td>MD5</td>\n",
       "      <td>PhiX control library was spiked in at 10%.</td>\n",
       "      <td>32PE | 320G</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>380</th>\n",
       "      <td>NGMT_GMT1_Sed_550m_10A_Extracted2306</td>\n",
       "      <td>Bacteria-16S-V4V5-Parada</td>\n",
       "      <td>GMT-2 Sed 16S Redo 2025-06-26</td>\n",
       "      <td>D1</td>\n",
       "      <td>4</td>\n",
       "      <td>GMT1_10A_dup_16S_S65</td>\n",
       "      <td>20250728_THO16883_16S-Amplicon_PE250</td>\n",
       "      <td>GMT1_10A_dup_16S_S65_L001_R1_001.fastq.gz</td>\n",
       "      <td>GMT1_10A_dup_16S_S65_L001_R2_001.fastq.gz</td>\n",
       "      <td>2852e9d675100a17471bd0d04dcbd07c</td>\n",
       "      <td>...</td>\n",
       "      <td>Michigan State University Research Technology ...</td>\n",
       "      <td>ILLUMINA</td>\n",
       "      <td>Illumina MiSeq [OBI_0002003]</td>\n",
       "      <td>MiSeq v2 500</td>\n",
       "      <td>ACACTGACGACATGGTTCTACA</td>\n",
       "      <td>TACGGTAGCAGAGACTTGGTCT</td>\n",
       "      <td>The Genomics Core performed secondary PCR usin...</td>\n",
       "      <td>MD5</td>\n",
       "      <td>PhiX control library was spiked in at 10%.</td>\n",
       "      <td>32PE | 320G</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>381</th>\n",
       "      <td>NGMT_GMT1_Sed_550m_10A_Extracted2506</td>\n",
       "      <td>Bacteria-16S-V4V5-Parada</td>\n",
       "      <td>GMT-6 Sed + GMT-7 CTD 16S 2025-06-26</td>\n",
       "      <td>D7</td>\n",
       "      <td>52</td>\n",
       "      <td>GMT1_10A_dup5_16S_S52</td>\n",
       "      <td>20250728_THO16883_16S-Amplicon_PE250</td>\n",
       "      <td>GMT1_10A_dup5_16S_S52_L001_R1_001.fastq.gz</td>\n",
       "      <td>GMT1_10A_dup5_16S_S52_L001_R2_001.fastq.gz</td>\n",
       "      <td>41e619ead2a35132ee2309cc443391b3</td>\n",
       "      <td>...</td>\n",
       "      <td>Michigan State University Research Technology ...</td>\n",
       "      <td>ILLUMINA</td>\n",
       "      <td>Illumina MiSeq [OBI_0002003]</td>\n",
       "      <td>MiSeq v2 500</td>\n",
       "      <td>ACACTGACGACATGGTTCTACA</td>\n",
       "      <td>TACGGTAGCAGAGACTTGGTCT</td>\n",
       "      <td>The Genomics Core performed secondary PCR usin...</td>\n",
       "      <td>MD5</td>\n",
       "      <td>PhiX control library was spiked in at 10%.</td>\n",
       "      <td>32PE | 320G</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>8 rows × 205 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                              sample_name                assay_name  \\\n",
       "27                 NGMT_GMT1_Sed_550m_10A  Bacteria-16S-V4V5-Parada   \n",
       "49   NGMT_GMT1_Sed_550m_10A_Extracted2306  Bacteria-16S-V4V5-Parada   \n",
       "50   NGMT_GMT1_Sed_550m_10A_Extracted2306  Bacteria-16S-V4V5-Parada   \n",
       "192  NGMT_GMT1_Sed_550m_10A_Extracted2306  Bacteria-16S-V4V5-Parada   \n",
       "193  NGMT_GMT1_Sed_550m_10A_Extracted2401  Bacteria-16S-V4V5-Parada   \n",
       "291  NGMT_GMT1_Sed_550m_10A_Extracted2412  Bacteria-16S-V4V5-Parada   \n",
       "380  NGMT_GMT1_Sed_550m_10A_Extracted2306  Bacteria-16S-V4V5-Parada   \n",
       "381  NGMT_GMT1_Sed_550m_10A_Extracted2506  Bacteria-16S-V4V5-Parada   \n",
       "\n",
       "                             pcr_plate_id pcr_well_position pcr_well_number  \\\n",
       "27                  GMT-1 Sed 16S 2022-09                D4              28   \n",
       "49                  GMT-2 Sed 16S 2023-08                D1               4   \n",
       "50               GMT-3 Sed 16S 2023-07-28                F6              46   \n",
       "192         GMT-2 Sed 16S Redo 2024-01-26                D1               4   \n",
       "193  GMT-4 Sed + GMT-5 CTD 16S 2024-01-25                C1               3   \n",
       "291              GMT-5 Sed 16S 2024-12-18                H8              64   \n",
       "380         GMT-2 Sed 16S Redo 2025-06-26                D1               4   \n",
       "381  GMT-6 Sed + GMT-7 CTD 16S 2025-06-26                D7              52   \n",
       "\n",
       "                    lib_id                            seq_run_id  \\\n",
       "27        GMT1_16S_10A_S28           20220912_16S-Amplicon_PE250   \n",
       "49     GMT1_dup_16S_10A_S4      20231110_AND14513_Amplicon_PE250   \n",
       "50   GMT1_dup2_16S_10A_S91      20231110_AND14513_Amplicon_PE250   \n",
       "192   GMT1_dup_16S_10A_S58           20240325_16S-Amplicon_PE250   \n",
       "193   GMT1_dup3_16S_10A_S3           20240325_16S-Amplicon_PE250   \n",
       "291  GMT1_10A_dup4_16S_S64           20250423_16S-Amplicon_PE250   \n",
       "380   GMT1_10A_dup_16S_S65  20250728_THO16883_16S-Amplicon_PE250   \n",
       "381  GMT1_10A_dup5_16S_S52  20250728_THO16883_16S-Amplicon_PE250   \n",
       "\n",
       "                                       filename  \\\n",
       "27        GMT1_16S_10A_S28_L001_R1_001.fastq.gz   \n",
       "49     GMT1_dup_16S_10A_S4_L001_R1_001.fastq.gz   \n",
       "50   GMT1_dup2_16S_10A_S91_L001_R1_001.fastq.gz   \n",
       "192   GMT1_dup_16S_10A_S58_L001_R1_001.fastq.gz   \n",
       "193   GMT1_dup3_16S_10A_S3_L001_R1_001.fastq.gz   \n",
       "291  GMT1_10A_dup4_16S_S64_L001_R1_001.fastq.gz   \n",
       "380   GMT1_10A_dup_16S_S65_L001_R1_001.fastq.gz   \n",
       "381  GMT1_10A_dup5_16S_S52_L001_R1_001.fastq.gz   \n",
       "\n",
       "                                      filename2  \\\n",
       "27        GMT1_16S_10A_S28_L001_R2_001.fastq.gz   \n",
       "49     GMT1_dup_16S_10A_S4_L001_R2_001.fastq.gz   \n",
       "50   GMT1_dup2_16S_10A_S91_L001_R2_001.fastq.gz   \n",
       "192   GMT1_dup_16S_10A_S58_L001_R2_001.fastq.gz   \n",
       "193   GMT1_dup3_16S_10A_S3_L001_R2_001.fastq.gz   \n",
       "291  GMT1_10A_dup4_16S_S64_L001_R2_001.fastq.gz   \n",
       "380   GMT1_10A_dup_16S_S65_L001_R2_001.fastq.gz   \n",
       "381  GMT1_10A_dup5_16S_S52_L001_R2_001.fastq.gz   \n",
       "\n",
       "                    checksum_filename  ...  \\\n",
       "27   d678bd9054c9e861f2e21f3b19f050d7  ...   \n",
       "49   3c544b57b6949fd3ca628f2596c03dc9  ...   \n",
       "50   4c762a1dafa1b6094495bf1c9ebcc76f  ...   \n",
       "192  404331aa25b1ac9374de1c00eac5838f  ...   \n",
       "193  f5481be9c865798c1a07e4f15329d1c8  ...   \n",
       "291  58e9115f5c418a301623e380546194b3  ...   \n",
       "380  2852e9d675100a17471bd0d04dcbd07c  ...   \n",
       "381  41e619ead2a35132ee2309cc443391b3  ...   \n",
       "\n",
       "                                   sequencing_location  platform  \\\n",
       "27   Michigan State University Research Technology ...  ILLUMINA   \n",
       "49   Michigan State University Research Technology ...  ILLUMINA   \n",
       "50   Michigan State University Research Technology ...  ILLUMINA   \n",
       "192  Michigan State University Research Technology ...  ILLUMINA   \n",
       "193  Michigan State University Research Technology ...  ILLUMINA   \n",
       "291  Michigan State University Research Technology ...  ILLUMINA   \n",
       "380  Michigan State University Research Technology ...  ILLUMINA   \n",
       "381  Michigan State University Research Technology ...  ILLUMINA   \n",
       "\n",
       "                       instrument       seq_kit         adapter_forward  \\\n",
       "27   Illumina MiSeq [OBI_0002003]  MiSeq v2 500  ACACTGACGACATGGTTCTACA   \n",
       "49   Illumina MiSeq [OBI_0002003]  MiSeq v2 500  ACACTGACGACATGGTTCTACA   \n",
       "50   Illumina MiSeq [OBI_0002003]  MiSeq v2 500  ACACTGACGACATGGTTCTACA   \n",
       "192  Illumina MiSeq [OBI_0002003]  MiSeq v2 500  ACACTGACGACATGGTTCTACA   \n",
       "193  Illumina MiSeq [OBI_0002003]  MiSeq v2 500  ACACTGACGACATGGTTCTACA   \n",
       "291  Illumina MiSeq [OBI_0002003]  MiSeq v2 500  ACACTGACGACATGGTTCTACA   \n",
       "380  Illumina MiSeq [OBI_0002003]  MiSeq v2 500  ACACTGACGACATGGTTCTACA   \n",
       "381  Illumina MiSeq [OBI_0002003]  MiSeq v2 500  ACACTGACGACATGGTTCTACA   \n",
       "\n",
       "            adapter_reverse  \\\n",
       "27   TACGGTAGCAGAGACTTGGTCT   \n",
       "49   TACGGTAGCAGAGACTTGGTCT   \n",
       "50   TACGGTAGCAGAGACTTGGTCT   \n",
       "192  TACGGTAGCAGAGACTTGGTCT   \n",
       "193  TACGGTAGCAGAGACTTGGTCT   \n",
       "291  TACGGTAGCAGAGACTTGGTCT   \n",
       "380  TACGGTAGCAGAGACTTGGTCT   \n",
       "381  TACGGTAGCAGAGACTTGGTCT   \n",
       "\n",
       "                                            lib_screen checksum_method  \\\n",
       "27   The Genomics Core performed secondary PCR usin...             MD5   \n",
       "49   The Genomics Core performed secondary PCR usin...             MD5   \n",
       "50   The Genomics Core performed secondary PCR usin...             MD5   \n",
       "192  The Genomics Core performed secondary PCR usin...             MD5   \n",
       "193  The Genomics Core performed secondary PCR usin...             MD5   \n",
       "291  The Genomics Core performed secondary PCR usin...             MD5   \n",
       "380  The Genomics Core performed secondary PCR usin...             MD5   \n",
       "381  The Genomics Core performed secondary PCR usin...             MD5   \n",
       "\n",
       "                          seq_method_additional ship_crs_expocode  \n",
       "27   PhiX control library was spiked in at 10%.       32PE | 320G  \n",
       "49   PhiX control library was spiked in at 10%.       32PE | 320G  \n",
       "50   PhiX control library was spiked in at 10%.       32PE | 320G  \n",
       "192  PhiX control library was spiked in at 10%.       32PE | 320G  \n",
       "193  PhiX control library was spiked in at 10%.       32PE | 320G  \n",
       "291  PhiX control library was spiked in at 10%.       32PE | 320G  \n",
       "380  PhiX control library was spiked in at 10%.       32PE | 320G  \n",
       "381  PhiX control library was spiked in at 10%.       32PE | 320G  \n",
       "\n",
       "[8 rows x 205 columns]"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "metadata[metadata['sample_name'].str.contains('NGMT_GMT1_Sed_550m_10A')]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "d525e7e2",
   "metadata": {},
   "outputs": [],
   "source": [
    "ex = 'ABCDEFGHIJ'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "9d2e382c",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'ABC'"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ex[:3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "e13a2f5b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'HIJ'"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ex[-3:]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "f0c5b5a2",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "pandas",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
